blockchain.js

```diff
@@ -37,7 +37,7 @@ function _setMainChain(isMainChain) {
   }
 }
 // _setMainChain(!/^test\./.test(location.hostname));
-_setMainChain(/^main\./.test(location.hostname));
+_setMainChain(true);
```

Comment:
Import the constant?

Reply:
added https://github.com/webaverse/app/pull/879/commits/0981f3fe49e92719fd2576cfa7eedc5e560c11b3